extension-manifest-v3/src/background/adblocker.js

```diff
@@ -295,3 +295,58 @@ chrome.runtime.onMessage.addListener((msg, sender) => {
 
   return false;
 });
+
+if (__PLATFORM__ === 'firefox') {
+  chrome.webRequest.onBeforeRequest.addListener(
+    (details) => {
+      const sourceUrl =
+        details.initiator || details.originUrl || details.documentUrl;
+      const request = Request.fromRawDetails({
+        _originalRequestDetails: details,
+        requestId: details.requestId,
+        tabId: details.tabId,
+        type: details.type,
+        url: details.url,
+        ...(sourceUrl ? { sourceUrl } : null),
+      });
+
+      if (pausedDomains.includes(request.domain)) {
+        return {};
+      }
+
+      if (request.type === 'other') {
```

Comment:
I am not sure if we need it. In `adblocker-extension` it is run only if the config is set correctly. I checked and our engines have it turned off.

https://github.com/ghostery/adblocker/blob/master/packages/adblocker-webextension/adblocker.ts#L529